### PR TITLE
K8SPG-145 PG operator 1.1.0 images

### PR DESCRIPTION
**Pull request by cap1984**
Since 1.0.0 PG operator does not support version service
we've decided to move legit configuration to 1.1.0 related files

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | failed |

**Review by tplavcic — DISMISSED** [review]

**Review by tplavcic — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | failed |

File: api-tests/apply_route_test.go [hunk 1]
```diff
@@ -114,12 +114,12 @@ func TestApplyPgShouldReturnSameMajorVersion(t *testing.T) {
...
 	pgParams := &version_service.VersionServiceApplyParams{
 		Apply:           "latest",
-		OperatorVersion: "1.0.0",
+		OperatorVersion: "1.1.0",
 		Product:         "pg-operator",
 	}
 	pgParams.WithTimeout(2 * time.Second)
 
-	for _, v := range []string{"12.8", "13.4"} {
+	for _, v := range []string{"12.8", "13.4", "14.1"} {
```
Comment (hors) [inline api-tests/apply_route_test.go:122]:
```suggestion
	for _, v := range []string{"12.8", "13.5", "14.1"} {
```
in 1.1.0 we will have PG 12.8 , 13.5 and 14.1 please fix PR according this information

**Review by hors — CHANGES_REQUESTED** [review]

File: api-tests/apply_route_test.go (hunk 1 above)
Comment (cap1984) [reply api-tests/apply_route_test.go]: done

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

**Review by tplavcic — DISMISSED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

**Review by hors — APPROVED** [review]

**Review by tplavcic — APPROVED** [review]

**Pull request merged**